.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,51 @@
+name: Bug report 🐞
+description: File a bug report
+title: "[Bug] "
+labels: bug
+body:
+- type: checkboxes
+  id: existing-issue
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
+- type: textarea
+  id: what-happened
+  attributes:
+    label: Describe the bug
+    description: A concise description of what you are experiencing.
+    placeholder: Tell us what you see!
+  validations:
+    required: true
+- type: textarea
+  id: expected-behaviour
+  attributes:
+    label: Expected behavior
+    description: A clear and concise description of what you expected to happen.
+  validations:
+    required: true
+- type: textarea
+  id: screenshots
+  attributes:
+    label: Add Screenshots
+    description: Add sufficient Screenshots to explain your issue.
+- type: dropdown
+  id: devices
+  attributes:
+    label: On which device are you experiencing this bug?
+    multiple: true
+    options:
+    - Linux
+    - Chrome
+    - Windows
```

Comment:
This list seems strange. You have 2 OSs and a browser. I think you should have:
- Linux
- Windows
- MacOS

But maybe it would be better to not have a dropdown list to ensure we aren't excluding anyone using a crazy OS no-one's heard off